server/embed/config.go

```diff
@@ -380,6 +382,9 @@ type Config struct {
 	// Defaults to 0.
 	ExperimentalDistributedTracingSamplingRatePerMillion int `json:"experimental-distributed-tracing-sampling-rate"`
 
+	EnableWatchDog                  bool          `json:"experimental-enable-watchdog"`
+	WatchDogTolerateInactiveTimeout time.Duration `json:"experimental-watchdog-tolerate-inactive-timeout"`
```

Comment:
1. We should be very explicit that it's about etcd-raft-loop watchdog, as there might be other watchdogs in the future. 
2. watchdog-timout should be sufficient to understand it's about being inactive.

Reply:
For # 1,  I thought it's etcd-raft-loop watchdog in the beginning, but seems not good on second thought. Please see https://github.com/etcd-io/etcd/pull/15440#issuecomment-1467334490.

For # 2, agreed.